src/diffusers/loaders.py

```diff
@@ -1622,7 +1671,11 @@ def load_lora_into_unet(
                 if "lora_B" in key:
                     rank[key] = val.shape[1]
 
-            lora_config_kwargs = get_peft_kwargs(rank, network_alphas, state_dict, is_unet=True)
+            if peft_config is not None:
+                lora_config_kwargs = LoraConfig.from_json_file(peft_config)
+            else:
+                lora_config_kwargs = get_peft_kwargs(rank, network_alphas, state_dict, is_unet=True)
+
```

Comment:
Shouldn't this come under an `if USE_PEFT_BACKEND`?

Reply:
Good point!